package.json

```
@@ -7,7 +7,8 @@
     "format-check": "prettier --check **/*.js",
     "lint": "eslint index.js",
     "package": "ncc build index.js -o dist",
-    "test": "eslint index.js && jest",
+    "test": "eslint index.js && jest --collect-coverage",
```

```suggestion
    "pretest": "npm run lint",
    "test": "jest --collect-coverage",
```

We should use npm's hooks to run the linter before the test are run. We should also reuse the lint npm script.

---

For reference https://docs.npmjs.com/misc/scripts